### ISSUE TITLE
Annotation search not working

### ISSUE THREAD

**Issue by osaze-nosaze** [elias-sundqvist/obsidian-annotator#174]
Hi,
Thanks for this great plugin. Could you look at the search feature above the annotations pane, though? Annotation search does not function anymore on my system: Ubuntu 20.04.4; Obsidian 0.14.6; Obsidian-Annotator 0.2.2. I type a string into the search bar, press <Enter>, but nothing happens.
I do not know whether this has to do with the failure of auto-update for the plugin a few days back. Annotator repeatedly would not update, although other plugins did. I had to download the Annotator zip file and unzip its contents into ".obsidian/plugins/obsidian-annotator". Everything but this annotation search feature seems to work fine so far.
Thanks for your help.

**Comment by aladmit** [comment]
Yep, I see the same problem on my vault. I found that this error is root cause, but didn't figure out yet why it appeared with latest release 🤔

> Blocked form submission to '#' because the form's frame is sandboxed and the 'allow-forms' permission is not set.

**Comment by huyz** [comment]
Are you talking about the magnifying glass icon; i.e. what I would call PDF search? (I'm confused why you're calling it the "Annotation search")

**Comment by connor-minton** [comment]
> Are you talking about the magnifying glass icon; i.e. what I would call PDF search? (I'm confused why you're calling it the "Annotation search")

@huyz 
Not the PDF search. The magnifying glass in the section that lists the annotations. When you hover over that icon, it says "Search annotations"

**Comment by AmaanAnsari** [comment]
Same issue here: Windows, Plugin Version: 0.2.7

**Comment by vatalaga** [comment]
Same issue here: Windows, Release V0.2.10

**Comment by dtonda8** [comment]
Issue is still here as of today.
One possible work around is to open the file back into mark down mode and then do a regular search.

**Comment by madukan** [comment]
It is this search. The one right above the annotations/pages section. Note the search box opened there in the image. This box opens up when you click the magnifying glass. I took the screenshot a little bit across the screen so the proximity of the search icon is clear. Hope others are talking the same. This search is important to search across the notes and annotations. 

<img width="589" alt="image" src="https://github.com/elias-sundqvist/obsidian-annotator/assets/7971561/215523cd-cdd7-489e-bf9c-9fdaf165966d">
